ree.coffee

```diff
@@ -0,0 +1,2 @@
+while "ree"
+  console.log "reee"
```

Comment:
This should print r and an infinite stream of e afterwards

Reply:
```coffee
console.log "r"
while "reeee"
 console.log "e"
```

I gues, never tested